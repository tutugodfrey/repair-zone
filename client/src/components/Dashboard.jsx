import React, { Component } from 'react';
import Div from './elementComponents/Div.jsx';
import Header from './Header.jsx';
import Footer from './Footer.jsx';
import Content from './Content.jsx';

export default class Dashboard extends Component {
  dashboardContent() {
    return (
      <div>
        <Header />
        <Content />
        <Footer />
      </div>
    )
  }
  render() {
    return <Div divId="" divClass="" content={this.dashboardContent()} />
  }
}