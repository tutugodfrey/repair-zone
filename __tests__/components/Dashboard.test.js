import React from 'react';
import { shallow} from 'enzyme';
import toJson from 'enzyme-to-json';
import Dashboard from '../../client/src/components/Dashboard.jsx';

describe('<Dashboard />', () => {
  const wrapper = shallow(
    <Dashboard />
  );
  test('should match the snapshot of the component', () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
