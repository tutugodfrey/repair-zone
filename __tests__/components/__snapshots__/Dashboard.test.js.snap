// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Dashboard /> should match the snapshot of the component 1`] = `
<Div
  content={
    <div>
      <Connect(Header) />
      <Connect(Content) />
      <Footer />
    </div>
  }
  divClass=""
  divId=""
/>
`;
